pubspec.yaml

```diff
@@ -11,14 +11,14 @@ platforms:
   windows:
 
 environment:
-  sdk: ">=2.17.0 <3.0.0"
-  flutter: ">=3.0.0"
+  sdk: ">=2.19.1 <3.0.0"
+  flutter: ">=3.10.1"
```

Comment:
Why you would like to force to flutter 3.10.1? I prefer to support the last 3 versions, you can bump to 3.3.0 at least